pkg/gcp/actuator/actuator.go

```diff
@@ -64,6 +64,10 @@ type Actuator struct {
 	GCPClientBuilder func(string, []byte) (ccgcp.Client, error)
 }
 
+func (a *Actuator) STSFeatureGateEnabled() bool {
```

Comment:
nit: I vaguely remember reading this concept being named something not AWS-specific (as opposed to STS) - does that matter?

Reply:
Actually GCP calls it STS too. So it's okay naming I think.